definitions.yaml

```diff
@@ -141,6 +147,12 @@ definitions:
         type: integer
         format: int32
         description: Expiration time (from creation) in hours
+      cn_prefix:
+        type: string
+        description: The common name prefix of the certificate subject.
+      o:
```

Comment:
Would it make sense to make our public API more elaborative? Don't know if this would be better UX than a single letter.